app/src/main/java/com/github/sdp/mediato/ui/MyProfileFragment.java

```diff
@@ -91,20 +89,19 @@ public CollectionListAdapter setupCollections(RecyclerView recyclerView) {
         // Check if a collection is already in the viewModel, if not create the default one
         List<Collection> collections = viewModel.getCollections();
         if (collections == null) {
-            String defaultTitle = getResources().getString(R.string.recently_watched);
-            Collection defaultCollection = new Collection(defaultTitle);
-            collections = new ArrayList<>();
-            collections.add(defaultCollection);
-            CollectionsDatabase.addCollection(USERNAME, defaultCollection);
-            viewModel.setCollections(collections);
+            collections = createDefaultCollection();
         }
 
         // Define what happens when the add button inside a collection is clicked
         OnAddMediaButtonClickListener onAddMediaButtonClickListener = (collection, review) -> {
-            CollectionsDatabase.addReviewToCollection(USERNAME, collection.getCollectionName(), review);
-            Collection currentCollection = viewModel.getCollection(collection.getCollectionName());
-            ((MyProfileViewModel)viewModel).addReviewToCollection(review, "sample collection");
-
+            String collectionName = collection.getCollectionName();
+
+            // Pass the name of the collection to add the review to to the search fragment and switch to it
+            SearchFragment searchFragment = new SearchFragment();
```

Comment:
When trying to add reviews directly on the app, I noticed that clicking on the button opens the search by default on the people category. Is there a way to make it by default on movies or something ? Maybe by adding a `perform(click())` on the movies button ?
Ideally we would also make the people button disappear when the search fragment is opened here. You tell me what is feasible in the scope of your PR but I think it shouldn't be hard to add.

Reply:
Yes I saw that this is not ideal. `Perfom(click())` only works for tests. We would probably need to pass an argument to the SearchFragment and change the view from there. But I think this should be done in another PR, as there are also other things that should be improved about the SearchFragment. For example, I think that the cursor should be placed in the search field by default as this is not very intuitive now. Also we need to remove the search category. This should probably be a task for next sprint.

Reply:
Alright I created a task for this. You can add another one for the cursor.